addons/product/static/src/js/pricelist_report/product_pricelist_report.js

```diff
@@ -183,6 +216,45 @@ export class ProductPricelistReport extends Component {
         });
     }
 
+    async export_pricelist_csv_xlsx(format) {
```

Comment:
Is it necessary to keep this method async ?
Can't we call the `download` and `doAction` without the `await` ?

Reply:
Yes, we can call the `download` and `doAction` without the await. We can remove the `async` and `await` if we don’t need to wait for the download to complete before moving on. What would you suggest ?

Reply:
I did believe it was probably unnecessary to use await here, but after talking about it with VCR, we're not fully sure and don't want to waste time to test that :D.
Let's keep it like this for now.